jenkins/update-jobs.sh

```diff
@@ -33,14 +44,13 @@ if ! docker inspect job-builder &> /dev/null; then
       --net host \
       --name job-builder \
       --restart always \
-      gcr.io/google_containers/kubekins-job-builder:3
+      -v "$(pwd):/test-infra" \
```

Comment:
I would have to do `$(dirname "${BASH_SOURCE}")/..`. Do you think that's better?